hyprpm/src/main.cpp

```diff
@@ -11,7 +11,7 @@
 
 const std::string HELP = R"#(┏ hyprpm, a Hyprland Plugin Manager
 ┃
-┣ add [url]              → Install a new plugin repository from git
+┣ add [url] [git rev]    → Install a new plugin repository from git. Git revision is optional, when set, commit locks are ignored.
```

Comment:
come on, you know full well the line is too long. Break it somewhere and align to `Install`

Reply:
was kinda hoping you'd just ignore that lol, sure done